Explorer/Assets/DCL/WebRequests/GenericDownloadHandlerUtils.cs

```diff
@@ -20,28 +20,6 @@ public static class GenericDownloadHandlerUtils
     {
         public delegate Exception CreateExceptionOnParseFail(Exception exception, string text);
 
-        public static Adapter<GenericPostRequest, GenericPostArguments> SignedFetchPostAsync(
```

Comment:
is it OK to remove this one?

Reply:
I just moved it inside another extension file for all SignedFetches